compiler/parser/parser.peg

```diff
@@ -962,6 +954,10 @@ ComparisonExpr
         Loc: loc(c),
       }, nil
     }
+ 
+ IsNull
+  = IS _ NULL { return "==", nil }
+  / IS _ NOT _ NULL { return "!=", nil }
 
```

Comment:
Should this be its own expression? Once we get to SQL semantics for null values `IS NULL` will behave differently than `foo == null`...